testworld/httputils.go

```diff
@@ -41,13 +42,30 @@ func createInsecureClientWithExpect(t *testing.T, baseURL string) *httpexpect.Ex
 	return httpexpect.WithConfig(config)
 }
 
-func getDocumentAndCheck(e *httpexpect.Expect, auth string, documentType string, params map[string]interface{}) *httpexpect.Value {
+func getDocumentAndCheck(t *testing.T, e *httpexpect.Expect, auth string, documentType string, params map[string]interface{}, checkattrs bool) *httpexpect.Value {
 	docIdentifier := params["document_id"].(string)
 
 	objGet := addCommonHeaders(e.GET("/"+documentType+"/"+docIdentifier), auth).
 		Expect().Status(http.StatusOK).JSON().NotNull()
 	objGet.Path("$.header.document_id").String().Equal(docIdentifier)
 	objGet.Path("$.data.currency").String().Equal(params["currency"].(string))
+	if checkattrs {
+		attrs := objGet.Path("$.data.attributes").Object().Raw()
+		eattrs := defaultAttributePayload()
```

Comment:
I would have passed it as an extra func parameter like `params` for the currency check but not so important.